src/components/analysis/viewParts/viewPartsButton.jsx

```diff
@@ -33,14 +34,7 @@ const ViewParts = ({ setFileFor3dModel }) => {
     fetch(`/api/v0/parts/${part_id}/file`)
       .then((res) => res.blob())
       .then((blob) => {
```

Comment:
I think that changing the name of downloadFile -> downloadFileToLocalMachine or something could be useful as well to differentiate it from the other function that downloads the file into a blob and uses it for the viewer